### PR TITLE
Bug 2040136: notice about the extra roles

### PR DESCRIPTION
**Pull request by alebedev87**
This PR adds a notice about the pre-requisite extra roles.
image

### CODE REVIEW

File: bundle/manifests/external-dns-operator_clusterserviceversion.yaml [hunk 1]
```diff
@@ -44,7 +44,11 @@ spec:
       kind: ExternalDNS
       name: externaldnses.externaldns.olm.openshift.io
       version: v1alpha1
-  description: ExternalDNS Operator deploys and manages ExternalDNS, which dynamically manages external DNS records in specific DNS Providers for specific Kubernetes resources.
+  description: "The ExternalDNS Operator deploys and manages ExternalDNS, which dynamically manages DNS records in external DNS Providers for specific Kubernetes resources.\n\n## Prerequisites and Requirements\n
+    The ExternalDNS Operator has minimal cluster level permissions. Unfortunately OLM doesn't allow the local role creation in the operand namespace.\n
+    Therefore make sure the following commands are run against your cluster before the installation of the operator:\n
+    * Create the operand namespace: `oc create ns external-dns`\n
+    * Grant operator the access to operand: `oc apply -f config/rbac/extra-roles.yaml`"
```
Comment (Miciah) [inline bundle/manifests/external-dns-operator_clusterserviceversion.yaml:51]: Is the user reading this notice in console going to know where to get the  `config/rbac/extra-roles.yaml` file?  Should we provide a full URL?  
```suggestion
    * Grant the operator access to manage operand resources: `oc apply -f https://raw.githubusercontent.com/openshift/external-dns-operator/main/config/rbac/extra-roles.yaml`"
```

**Review by sherine-k — COMMENTED** [review]
Thanks @alebedev87 ! this will drastically clarify.
Do you think I should add the `create ns` inside the extra-roles.yml file? 
Do you think it sounds as a "promise" if we add the "yet" to the OLM handling creation of resources in the operand ns?

**Comment by Miciah** [comment]
Thanks!
/lgtm

**Pull request merged**